.circleci/config.yml

```diff
@@ -17,6 +17,15 @@ defaults:
         [ -n "$COVERAGE" -a "$CIRCLE_BRANCH" != release -a -z "$CIRCLE_TAG" ] && CMAKE_OPTIONS="$CMAKE_OPTIONS -DCOVERAGE=ON"
         cmake .. -DCMAKE_BUILD_TYPE=Release $CMAKE_OPTIONS
         make -j4
+  - run_build_ossfuzz: &run_build_ossfuzz
+      name: Build_ossfuzz
+      command: |
+        mkdir -p build
+        cd build
+        rm -rf *
+        /src/LPM/external.protobuf/bin/protoc --proto_path=../test/tools/ossfuzz yulProto.proto --cpp_out=../test/tools/ossfuzz
+        CXX_FLAGS="-O1 -gline-tables-only -fsanitize=address -fsanitize-address-use-after-scope -fsanitize=fuzzer-no-link -stdlib=libstdc++" cmake .. -DCMAKE_BUILD_TYPE=Release $CMAKE_OPTIONS
```

Comment:
Can you use a cmake option directly for these settings?

Reply:
alternatively, I think it would be better to turn these into environment variables like in the `build_x86_clang7_asan` job.

Reply:
The problem is double quotes within a cmake option is not parsed correctly. I spent some time experimenting with bash alternatives but couldn't find an elegant solution.

Reply:
I don't understand - why do you need double quotes?
You can make CXX_FLAGS an environment variable just like in 
https://github.com/ethereum/solidity/pull/6413/files#diff-1d37e48f9ceff6d8030570cd36286a61R144

Reply:
I wasn't aware of custom cmake files until now :) Updated the PR likewise